### ISSUE TITLE
I feel there is a room for improvement on the header;

### ISSUE THREAD

**Issue by lykmapipo** [CodeTanzania/emis-dashboard#30]
![screenshot from 2018-09-13 20-01-18](https://user-images.githubusercontent.com/1610857/45503558-dfef2c00-b78f-11e8-83b0-e9f97b43d58d.png)

**Comment by kasongoyo** [comment]
I agree. I also felt something was not right about the action button and the header in a comparison to other pages
